packages/console/src/components/ImagePlaceholder/index.tsx

```diff
@@ -14,11 +14,11 @@ const ImagePlaceholder = ({ size = 40, borderRadius = 8 }: Props) => {
   const { t } = useTranslation();
 
   return (
-    <div className={styles.container} style={{ width: size, height: size, borderRadius }}>
+    <div className={styles.container} style={{ borderRadius }}>
       <img
         alt={t('general.placeholder')}
         src={defaultPlaceholder}
-        style={{ width: size, height: size }}
+        style={{ width: size - 16, height: size - 16 }}
```

Comment:
Can we use CSS instead of inline styles? Consider exposing a `className` property from this `ImagePlaceholder` component and allow the caller component to overwrite default styles

Reply:
Currently, we are using a temp picture, so for now, this PR is just a fix, we can change this logic when the final picture comes out.